installer/conf/container.conf

```diff
@@ -23,6 +23,14 @@
   log_level debug
 </source>
 
+#custom_metrics_mdm filter plugin
+<filter mdm.cadvisorperf**>
+  type filter_cadvisor2mdm
+  custom_metrics_azure_regions eastus,southcentralus,westcentralus,westus2,southeastasia,northeurope,westEurope
```

Comment:
Not right now, but this needs to go outside the plugin config to seperate config so that others can use it and we dont need to have multiple copies. Please log a workitem to fix for the future